sources/Models/Vector3.hpp

```diff
@@ -158,6 +158,22 @@ namespace BBM
 		{
 			return RAY::Vector3(this->x, this->y, this->z);
 		}
+
+		static Vector3<T> min(Vector3<T> a, Vector3<T> b)
```

Comment:
you should check with zoe to be able to enable man and max function only with floats or  types that supports std::min/std::max

Reply:
@AnonymusRaccoon @GitBluub didn't had the mentions in clion integration